pkgs/development/python-modules/djhtml/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "djhtml";
+  version = "3.0.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-84/U59V1U4snkcXKorVMU5etwI+PjSZYU+Ha+oojmuk=";
+  };
+
```

Comment:
You should add an import check

Reply:
It is not really a module that would imported into python code, it's more like black and gives a script to format files

Reply:
Then, it should not be a Python application

Reply:
The `pythonImportsCheck` would still work or at least it does most of the time.

If the primary use case is not as module then use `toPythonApplication` in `all-packages.nix`.

Reply:
Done